core-java/pom.xml

```diff
@@ -162,6 +162,12 @@
             <version>${avaitility.version}</version>
             <scope>test</scope>
         </dependency>
+		<dependency>
+		    <groupId>org.springframework</groupId>
```

Comment:
Why do we need spring in the core-java module?

Reply:
It is one of the libraries that is used to show how to delete directories recursively in Java, and that article and code is part of java-core....